wormhole_chain/app/app.go

```diff
@@ -383,11 +405,52 @@ func New(
 		&stakingKeeper, govRouter,
 	)
 
+	// The last arguments can contain custom message handlers, and custom query handlers,
+	// if we want to allow any custom callbacks
+	supportedFeatures := "iterator,staking,stargate"
+	wasmDir := filepath.Join(homePath, "data")
+	// Instantiate wasm keeper with stubs for other modules as we do not need
+	// wasm to be able to write to other modules.
+	app.wasmKeeper = wasm.NewKeeper(
+		appCodec,
+		keys[wasm.StoreKey],
+		app.GetSubspace(wasm.ModuleName),
+		// &wasmHandlers.AccountKeeperHandler{},
```

Comment:
why is this commented out?

Reply:
I wrapped all the keepers to wasmd to prevent any write access to other modules, so instead this is passed.

```
&wasm_handlers.AccountKeeperHandler{AccountKeeper: app.AccountKeeper},
```

In hindsight this isn't necessary as all the account methods are passed through anyways. (wasmd creates an account when a contract is created).